src/utils/JWTs.ts

```diff
@@ -0,0 +1,25 @@
+import jwt from "jsonwebtoken"
+import {secrets} from "./secrets";
+
+const JWTSecret = secrets.JWTSecret || "test"
```

Comment:
Do we have only one Secret? I don't know exactly how I would justify it, but doesn't a keypair provide more security?

Reply:
🤷